### PR TITLE
fix: added convinience methods and arguments for fetching projects

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

File: modelon/impact/client/client.py [hunk 1]
```diff
@@ -258,7 +258,7 @@ def get_workspaces(self):
             for item in resp["data"]["items"]
         ]
 
-    def get_project(self, project_id):
+    def get_project(self, project_id: str, vcs_info: bool = True):
```
Comment (efredriksson-modelon) [inline modelon/impact/client/client.py:261]: Should default be `True`?

File: modelon/impact/client/entities/workspace.py [hunk 2]
```diff
@@ -157,6 +157,9 @@ def id(self) -> str:
         """Workspace id"""
         return self._workspace_id
 
+    def definition(self):
```
Comment (efredriksson-modelon) [inline modelon/impact/client/entities/workspace.py:160]: property?

**Review by efredriksson-modelon — CHANGES_REQUESTED** [review]
Looks ok from what I can see. 👍 Do we want `vcs_info` `True` by default?

File: modelon/impact/client/client.py (hunk 1 above)
Comment (abhilash-kumar-nair) [reply modelon/impact/client/client.py:261]: Yes, we agreed on having this True in the group chat right?

Comment (iakovn) [reply modelon/impact/client/client.py:261]: From me trying the client out - yes, True is preferable/safer default. This is mostly to give expected result for someone not reading the docs before trying out the code.

**Review by iakovn — APPROVED** [review]
Looking good! 👍

**Pull request merged**